gensim/api/__init__.py

```diff
@@ -14,23 +16,52 @@
 except ImportError:
     from urllib2 import urlopen
 
+logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', filename="api.log", level=logging.INFO)
+console = logging.StreamHandler()
+console.setLevel(logging.INFO)
+logging.getLogger('').addHandler(console)
+
 
-def download(file):
+def download(file_name):
     url = "https://github.com/chaitaliSaini/Corpus_and_models/releases/"
-    url = url+"download/"+file+"/"+file+".tar.gz"
+    url = url + "download/" + file_name + "/" + file_name + ".tar.gz"
     user_dir = os.path.expanduser('~')
     base_dir = os.path.join(user_dir, 'gensim-data')
-    extracted_folder_dir = os.path.join(base_dir, file)
+    extracted_folder_dir = os.path.join(base_dir, file_name)
     if not os.path.exists(base_dir):
+        logging.info("Creating {}".format(base_dir))
```

Comment:
Module should create a `logger` and use that for logging (not these global `logging` wrappers). See other gensim modules for an example.